source/reference/tutorial-rating.rst

```diff
@@ -0,0 +1,53 @@
+.. _tutorial-rating:
+
+Tutorial difficulty ratings
+###########################
+
+Tutorial difficulty ratings provide a simple way to start developing with 
+|CL-ATTR|. If you're new to the distro, we suggest starting with ``Easy`` tutorials and  working towards the more ``Difficult``. Ratings not only expose learning paths but also provide springboards from which to advance or improve use cases, so be sure to :ref:`share your insights <collaboration>`. Three main metrics help us to determine how to rate a tutorial: 
```

Comment:
Suggest replacing "springboards" with "a starting point" or a similar more common phrasing.

Reply:
Yes. That would be difficult to translate. Unless you're a trapeze artist. :)